src/features/api/apiSlice.ts

```diff
@@ -1,12 +1,20 @@
 import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
 
 const baseQuery = fetchBaseQuery({
-  baseUrl: "" // add base API url when we start working with it
+  baseUrl: "http://localhost:5000/conda-store/api/v1",
```

Comment:
Should I place this URL in the .env variable?

Reply:
@ostojics  in the .env for the moment please.